apps/Obsidian/description

```diff
@@ -0,0 +1,13 @@
+Obsidian is a powerful knowledge base on top of a local folder of plain text Markdown files (its a note taking tool and markdown editor)
+
+Note-taking is incredibly personal. Tried every app, but something always irks you? You deserve better.
+For those who can't custom-build a solution for themselves, Obsidian is as close as you can get, with plugins, themes, and custom CSS at your disposal. 
+
+With Obsidian, your data sits in a local folder. Never leave your life's work held hostage in the cloud again.
+
+Vibrant Community
+With more than 50,000 members in our Discord chat and 30,000 members on our forum, Obsidian has one of the largest note-taking communities.
+Our community welcomes anyone who uses Obsidian or is interested in Obsidian, no matter your language, country, or field. Come join us!
+
+To run from Menu (after first launch): Menu -> Office -> Obsidian
```

Comment:
no this isn't the case
who wrote this? if the user has appimagelauncher this is the case, yes, but that isn't installed by default nor is it really the best thing honestly since it runs as a service waiting for any and all appimage launch attempts